rpcs3/Json/tooltips.json

```diff
@@ -86,6 +86,7 @@
 			"gs_disableMouse": "Disables the activation of fullscreen mode per double-click while the game screen is active.\nCheck this if you want to play with mouse and keyboard (for example with UCR).",
 			"maxLLVMThreads": "Limits the maximum number of threads used for PPU Module compilation.\nLower this in order to increase performance of other open applications.\nThe default uses all available threads.",
 			"showShaderCompilationHint": "Show shader compilation hints using the native overlay.",
+			"checkUpdatesOnStart": "If on the emulator will check if an update is available on launch",
```

Comment:
This sounds weird.

Reply:
Yeah, changing it, it was a last minute addition.

Reply:
If you mean the description, then it's because of a missing comma after the `on`.